molgenis-js/src/test/java/org/molgenis/js/JsMagmaScriptEvaluatorTest.java

```diff
@@ -155,7 +161,7 @@ public void testValueForNestedXrefDefaultDepth()
 
 		Object scriptExceptionObj = jsMagmaScriptEvaluator.eval("$('gender.xref.label').value()", person);
 		assertEquals(scriptExceptionObj.toString(),
-				"org.molgenis.script.core.ScriptException: <eval>:502 TypeError: Cannot read property \"label\" from undefined");
+				"javax.script.ScriptException: TypeError: Cannot read property \"label\" from undefined in <eval> at line number 431");
```

Comment:
Why expose internal ScriptException instead of our ScriptException that doesn't depend on Java scripting API?

Reply:
It was getting doubly wrappered and I didn't want to add a third wrapper. I've unwrapped it instead now.